_extensions/custom_roles.py

```diff
@@ -8,6 +8,12 @@
 
 
 def setup(app):
+    # role to link to cpython files
+    app.add_role(
+        "cpy-file",
```

Comment:
Using `gh-file` will make it more symmetric with `gh-label`, but since the two are not used together, I thought using `cpy-file` would make the fact that they are linking to the `cpython` repo more explicit.